docs/book/src/user/quick-start.md

```diff
@@ -93,6 +93,56 @@ a target [management cluster] on the selected [infrastructure provider].
    Then follow the instruction for your kind version using  `kind create cluster --config kind-cluster-with-extramounts.yaml`
    to create the management cluster using the above file.
 
+   {{#/tab }}
+   {{#tab KubeVirt}}
+
+   #### Create the Kind Cluster
+   [KubeVirt][KubeVirt] is a cloud native virtualization solution. The virtual machines we're going to create and use for
+   the workload cluster's nodes, are actually running within pods in the management cluster. In order to communicate
+   the workload cluster's API server, we'll need to expose it. We are using Kind which is a limited environment. The
+   easiest way to expose the workload cluster's API server (a pod within a node running in a VM that is itself running
+   within a pod in the management cluster, that is running inside a docker container), is to use a LoadBalancer service.
+
+   To allow using a LoadBalancer service, we can't use the kind's default CNI (kindnet), but we'll need to install
+   another CNI, like Calico. In order to do that, we'll need first to initiate the kind cluster with two modifications:
+   1. Disable the default CNI
+   2. Add the docker credentials to the cluster, to avoid the docker hub pull rate limit.
```

Comment:
How frequently does this happen? Are the kubevirt images in docker hub?
Asking because if it's not always necessary, we can omit it and maybe add it as troubleshooting step/advice. It adds one more thing the user needs to do and since this is a quick start, the less steps, the better.

Reply:
@g-gaston, this is pretty frequent. I would leave it here because (a) we need the configuration file anyway to disable kindnet CNI, and (b) because I think it's more than a regular troubleshoot, because starting the kind cluster is a basic pre-requisite. To re-do it, you must start over the whole process. So I think we need the right setting in advance.